src/pm-rpc/privates/__tests__/messageManagerSpec.js

```diff
@@ -0,0 +1,38 @@
+import _ from 'lodash'
+import {send} from '../messageManager'
+
+function createTarget() {
+  const {port1, port2} = new MessageChannel()
+  port1.onmessage = _.noop
+  spyOn(port2, 'postMessage').and.callThrough()
+  return port2
+}
+
+async function sendWithResolve(message, ...rest) {
+  // `send` sets `.__port` property
+  const sendPromise = send(message, ...rest)
+
+  message.__port.postMessage({intent: 'resolve', result: 'b206275b931a'})
+
+  return sendPromise
+}
```

Comment:
I can't get `send` to resolve 😫

Reply:
screw that! we’re doing `Promise.race`! https://github.com/wix/pm-rpc/pull/25/commits/4944d1fd22ceef35908046daba2c8892d34220ad